openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -149,9 +149,7 @@ def _configure_saver_tool(self, data, tool, subset):
 
         # get frame padding from anatomy templates
         anatomy = Anatomy()
-        frame_padding = int(
-            anatomy.templates["render"].get("frame_padding", 4)
-        )
+        frame_padding = anatomy.templates["frame_padding"]
```

Comment:
are we sure this is always returning intager? @iLLiCiTiT

Reply:
Good question. Just validated to be sure and in both OpenPype and AYON it does return `int`.